test/visual/backgrounds.js

```diff
@@ -22,15 +20,19 @@ async function createScreenshot(effectName) {
 async function testBackground(t, effect) {
   await createScreenshot(effect);
 
-  const [actual, expected] = await Promise.all([
-    readPNG(`${tempDir}/${effect}.png`),
-    readPNG(`${fixturePath}${effect}.png`)
-  ]);
```

Comment:
I see what went wrong here.  `readPNG` wasn't resolving to a value at all.  It's interesting that this failed quietly.

If you wanted to stick with the `Promise.all` approach I _think_ you'd just need to fix `readPNG`, which would look something like this:

```js
function readPNG(pngPath) {
  return new Promise(resolve => {
    const stream = fs.createReadStream(pngPath)
      .pipe(new PNG())
      .on('parsed', () => resolve(stream));
  });
}
```